mattermost-helm/values.yaml

```diff
@@ -151,15 +159,19 @@ mattermost-push-proxy:
     android_rn:
       apiKey: ""
 
+# Minio access/secret keys for the in-cluster S3 usage
 # See https://github.com/kubernetes/charts/tree/master/stable/minio#configuration
 # for more configuration options
+# Minio is not exposed publically
 minio:
-  accessKey: "AKIAIOSFODNN7EXAMPMM"
-  secretKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKMM"
-
+  enabled: true
+  imageTag: RELEASE.2018-05-25T19-49-13Z
+  serviceType: ClusterIP
+  accessKey: mattermostadmin
```

Comment:
Was this intentional?

Reply:
The values for the keys? Yes, replaced for some standard nomenclature. And also set the current minio release we are using